scripts/show-cf-memory-usage.rb

```diff
@@ -76,3 +76,4 @@ def format_memory(amount)
 puts "Memory reserved by apps: #{format_memory(apps_reserved_memory)}"
 puts
 puts "Total cell memory required to meet ADR017: #{format_memory(required_cell_memory)}"
+puts "Compare this against: https://app.datadoghq.com/metric/explorer?tile_size=s&exp_agg=sum&exp_metric=system.mem.total&exp_scope=bosh-job%3Acell&exp_group=bosh-deployment"
```

Comment:
Any reason to use this instead of https://metrics.cloud.service.gov.uk/dashboard/file/cell-memory-usage.json ? I find that Datadog page pretty confusing because it's got graphs for all the deployments.